packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 4.2.0 - Dec 20, 2022
+
+New features:
+
+- Use the new Dashboard tab template to create a personal and collaborative canvas in Teams. Learn more about this new template in the [documentation](https://aka.ms/teamsfx-dashboard-app).
+
+Enhancements:
+- Step-by-step guidance in Visual Studio Code to assisit you while creating a free Microsoft 365 testing tenant.
+- SPFx based tab app is updated to use the latest [SharePoint Framework v1.16](https://learn.microsoft.com/sharepoint/dev/spfx/release-1.16).
```

Comment:
"Scaffolds that use SharePoint Framework (SPFx) have been updated to use [SharePoint Framework version 1.16](https://learn.microsoft.com/sharepoint/dev/spfx/release-1.16) which introduces the ability to build experiences for Outlook and Office.com (Microsoft 365 app) in addition to Teams.

Reply:
Updated